### PR TITLE
Docker for testing

### PR DESCRIPTION
**Pull request by junderw**
run

```
docker run -p 127.0.0.1:49392:49392 junderw/btcpay-client-test-server
```

Then wait for the logs to die down.

Then run the tests.

We no longer need to use the public testnet instance.

### CODE REVIEW

**Comment by wallopthecat** [comment]
Very cool, love the puppeteer script, thanks

File: tests/core/client.spec.ts [hunk 1]
```diff
@@ -1,61 +1,99 @@
...
+        page,
+        browser,
+        pairingCode: '',
+      };
+    }
+    throw e;
+  }
 
   await page.type('#Email', USER_NAME);
   await page.type('#Password', PASSWORD);
```
Comment (takinbo) [inline tests/core/client.spec.ts:83]: The login screen is already pre-populated with these login credentials so entering the username and password again will append to those values and lead to a failed login. See attached screenshot.
loggedin

**Review by takinbo — CHANGES_REQUESTED** [review]

File: tests/core/client.spec.ts (hunk 1 above)
Comment (junderw) [reply tests/core/client.spec.ts]: Done

**Review by takinbo — APPROVED** [review]
tACK

**Pull request merged**